bcp-003-03-certificate-provisioning.md

```diff
@@ -238,14 +284,28 @@ On start up or on change of network state the EST Client MUST attempt to discove
 
 ### Certificate Revocation
 
-The EST Client SHOULD periodically check the revocation status of both the Root CA and their TLS Certificates using [OCSP][RFC-6960] and [CRL][RFC-5280]. If a TLS Certificate is revoked, the EST Client MUST stop using the revoked certificate immediately and follow [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow to replace the certificate.
+An EST Client SHOULD periodically check the revocation status of both the Root CA and their TLS Certificates using [OCSP][RFC-6960] and [CRL][RFC-5280]. If a TLS Certificate is revoked, the EST Client MUST stop using the revoked certificate immediately and follow [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow to replace the certificate.
 
 ## TODO:
-
-- Support for EST Server side generation of keys?
-  - EST can optionally support EST Server side generation of keys
-  - For low power devices, this could be useful, but should only be added if really necessary as not supported by many EST Servers
 - Consider using using TLS Client Certificates when using NMOS API, for use with BCP-003-02 OAuth
+- Consider including unique information in the manufacturer client certificate, such as MAC address, so the EST Server can validate that the client certificate presented was for the device in question (eg. the firmware has not been loaded onto a fake device)
+- Use of 802.1AR - secure device identity for a distinct PKI for client Auth
+- Define EST Client support for the CSR endpoint on the EST Server
+- Define the use of Root CA Key Update, for Root CA rollover/renewal
+- Possible extensions to EST RFC
+  - Add method to signal certificate profile, eg. TLS Client Auth or TLS Server Auth
+  - Add method to signal profile for server side key generation, digital signature algorithm
+  - discover of EST API
+
+## Security Considerations
+
+Boostrap Distribution of CA Certificate
+- It has been decided that EST Clients will implicitly trust EST Server's found using DNS-SD, no authentication of its TLS Certificate automatic or manual, needs to be performed.
+- However EST Clients MUST use the CA returned by the EST Server endpoint `/cacerts` to establish an Explicit Trust Anchor database used to for subsequent TLS authentication of the EST Server.
+- This deviation from the EST specification has been allowed due to the minimal risk around trusting a rogue EST Server
+  - If an EST Client is provisioned an incorrect CA Certificate by the rogue EST Server the EST Client would trust NMOS Node or NMOS registries. It would not divulge sensitive information, but quite clearly not work.
```

Comment:
Not sure exactly what this is trying to say, but there are some typos at least...

```suggestion
  - If an EST Client is provisioned with an incorrect CA Certificate by the rogue EST Server, the EST Client would trust (some? rogue?) NMOS Nodes or NMOS Registries. It would not divulge sensitive information, but(?) quite clearly not work.
```

Reply:
That section is still in progress -- we should review when done.